cluster-logging/eo-og.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: openshift-operators-redhat
+  namespace: openshift-operators-redhat 
+spec: {}
```

Comment:
I followed the instructions on https://docs.openshift.com/container-platform/4.2/logging/cluster-logging-deploying.html

Reply:
instead of the instructions, I read the description in the operator directly and I saw that sentence. Technically, it shouldn't matter which namespace we will use for the installation, but It would be maybe good to follow the approach of the OLM - install it into the default namespace. WDYT?

Reply:
actually, the [deployment instructions doc](https://docs.openshift.com/container-platform/4.2/logging/cluster-logging-deploying.html) says:  
> You must specify the openshift-operators-redhat namespace

Reply:
hmm... as I said - technically, it shouldn't matter in which namespace the operator is installed, so let's keep it as it is :-)